docs/public/_redirects

```diff
@@ -321,6 +324,11 @@ https://v4.material-ui.com/* https://v4.mui.com/:splat 301!
 /company/about/ /about/ 301
 /company/jobs/ /careers/ 301
 /material-ui/customization/unstyled-components/ /base/getting-started/overview/ 301
+/material/* /material-ui/:splat 301
+/zh/material/* /zh/material-ui/:splat 301
+/pt/material/* /pt/material-ui/:splat 301
```

Comment:
We used to link /material/ pages from the main domain, even if not predominantly, Google found them.